### PR TITLE
Refine description of Cinder error handling (SCRD-7699)

### PR DESCRIPTION
**Pull request by cadenzajon**

### CODE REVIEW

File: xml/operations-troubleshooting-ts_blockstorage.xml [hunk 1]
```diff
@@ -379,10 +356,19 @@ ansible-playbook -i hosts/verb_hosts cinder-reconfigure.yml</screen>
...
-   the log files. If it is running okay, check if the HAProxy service is
-   functioning properly.
+   If the API is inaccessible, determine if the API service has halted on
+   the controller nodes. (If a single instance of <literal>cinder-api</literal>
+   goes down but other instances remain online on other controllers, load
+   balancing would typically automatically direct all traffic to the online
+   nodes.) The <filename>cinder-status.yml</filename> playbook can be used to
+   report on the health of the API service from the &clm;:
+  </para>
+  <screen>cd ~/cratch/ansible/next/ardana/ansible
```
Comment (bryanstephenson) [inline xml/operations-troubleshooting-ts_blockstorage.xml]: Change "cratch" to "scratch".

File: xml/operations-troubleshooting-ts_blockstorage.xml [hunk 2]
```diff
@@ -315,36 +315,13 @@ ansible-playbook -i hosts/verb_hosts _cinder_post_check.yml</screen>
...
-  </para>
   <para>
    When creating your disk model for nodes that will have the cinder volume
-   role make sure that there is sufficient disk space allocated for a temporary
+   role, make sure that there is sufficient disk space allocated for temporary
    space for image conversion if you will be creating bootable volumes. You
-   should allocate enough space to the filesystem as would be needed to cater
-   for the raw size of images to be used for bootable volumes - for example
-   Windows images can be quite large in raw format.
+   should allocate enough space to the filesystem as would be needed to contain
```
Comment (asettle) [inline xml/operations-troubleshooting-ts_blockstorage.xml]: Remove "you should" and just have: "Allocate enough space to the filesystem to contain the raw..."

File: xml/operations-troubleshooting-ts_blockstorage.xml [hunk 3]
```diff
@@ -315,36 +315,13 @@ ansible-playbook -i hosts/verb_hosts _cinder_post_check.yml</screen>
...
+   the raw size of images to be used for bootable volumes. (For example,
+   Windows images can be quite large in raw format.)
```
Comment (asettle) [inline xml/operations-troubleshooting-ts_blockstorage.xml]: Also probably don't need this in paranthesis.

File: xml/operations-troubleshooting-ts_blockstorage.xml [hunk 4]
```diff
@@ -379,10 +356,19 @@ ansible-playbook -i hosts/verb_hosts cinder-reconfigure.yml</screen>
    <emphasis role="bold">API-level failures</emphasis>
   </para>
   <para>
-   If the API is inaccessible, determine if the API service is running on the
-   target node. If it is not, check to see why the API service is not running in
-   the log files. If it is running okay, check if the HAProxy service is
-   functioning properly.
+   If the API is inaccessible, determine if the API service has halted on
+   the controller nodes. (If a single instance of <literal>cinder-api</literal>
```
Comment (asettle) [inline xml/operations-troubleshooting-ts_blockstorage.xml]: ditto for the brackets

File: xml/operations-troubleshooting-ts_blockstorage.xml [hunk 5]
```diff
@@ -690,7 +677,7 @@ Request to delete volume 14b76133-e076-4bd3-b335-fa67e09e51f6 has been accepted.
 2016-04-27 12:39:15.567 6689 DEBUG swiftclient [req-0c65ec42-8f9d-430a-b0d5-05446bf17e34 - - -</screen>
   <para>
    If you determine that the volume is genuinely stuck in
-   <literal>detaching</literal> then you must follow the procedure described in
+   <literal>restoring</literal> then you must follow the procedure described in
```
Comment (asettle) [inline xml/operations-troubleshooting-ts_blockstorage.xml]: stuck in restoring? Like, a mode? If so, say, "... is stuck in retoring mode, then follow the procedure..."

**Pull request merged**